src/kudu/consensus/leader_election.cc

```diff
@@ -1314,6 +1328,8 @@ void LeaderElection::CheckForDecision() {
 
       result_.reset(new ElectionResult(
             request_, decision, highest_voter_term_, msg, is_candidate_removed));
+      if (vote_logger_)
+        vote_logger_->logElectionDecided(*result_.get());
```

Comment:
```suggestion
        vote_logger_->logElectionDecided(*result_);
```

Shouldn't need .get() for a shared_ptr/unique_ptr I think?